examples/math/render_primitives.rs

```diff
@@ -626,7 +648,7 @@ fn draw_gizmos_3d(mut gizmos: Gizmos, state: Res<State<PrimitiveSelected>>, time
                 .resolution(resolution),
         ),
         PrimitiveSelected::Ellipse => {}
```

Comment:
It looks like `Ellipse` was missed.

Reply:
I think ellipses are only missing from 3d gizmos and meshes, where the more natural thing would be ellipsoids (paralleling circle/sphere), which we don't have yet.